plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/JdkLdapAuthenticatorClient.java

```diff
@@ -149,6 +155,31 @@ private CloseableContext createUserDirContext(String userDistinguishedName, Stri
         return environment.build();
     }
 
+    private static SSLContext createIgnoreCertificateSslContext()
```

Comment:
Can we somehow reuse: io.trino.client.OkHttpUtil#setupInsecureSsl?

Reply:
I found that `trino-client` and `trino-password-authenticators` both use `trino-spi` but I think that `trino-spi` is not suitable this logic.